k8s/factory_mock.go

```diff
@@ -0,0 +1,21 @@
+package k8s
+
+import (
+	"github.com/play-with-docker/play-with-docker/pwd/types"
+	"github.com/stretchr/testify/mock"
+	"k8s.io/client-go/kubernetes"
+)
+
+type FactoryMock struct {
+	mock.Mock
+}
+
+func (m *FactoryMock) GetForSession(session *types.Session) (*kubernetes.Clientset, error) {
```

Comment:
This is not part of the interface, I think it might be a mistake and this was supposed to be `GetKubeletForInstance`?

Reply:
Right